### PR TITLE
PR for mailview indiv loading indicator and image loading for uploaded mp

### PR DESCRIPTION
**Pull request by sHuber125**
added circProgIndicator and updated mailpiece picture loading

### CODE REVIEW

File: lib/ui/mail_view_indv.dart [hunk 1]
```diff
@@ -40,23 +41,42 @@ class MailPieceViewWidgetState extends State<MailPieceViewWidget> {
...
+        reminderLinkUrl = null;
+        learnMoreLinkUrl = null;
+        hasDoMore = false;
+        hasLearnMore = false;
+        loading = false;
+      });
+    }
+    /*
+    setState(() {
+      loading = false;
```
Comment (AliFahimi) [inline lib/ui/mail_view_indv.dart]: Please remove all commented out code (unless we will need it later)

**Review by AliFahimi — DISMISSED** [review]
Looks good

File: lib/ui/mail_view_indv.dart (hunk 1 above)
Comment (mdconatser) [reply lib/ui/mail_view_indv.dart]: removed

**Review by mdconatser — DISMISSED** [review]

**Comment by mdconatser** [comment]
hijacked this PR to make a few changes:

- 90 days of email is now pulled rather than 30 to increase our test data
- Renamed app to MailSpeak
- Hid times from search/mail displays since it doesn't really matter and adds clutter for users with accessibility features
- Simplified mail piece display title

**Review by mdconatser — APPROVED** [review]

**Review by shaneknows — APPROVED** [review]

**Pull request merged**